packages/react/src/components/TableOfContents/TOCMobile.js

```diff
@@ -39,8 +39,31 @@ const TOCMobile = ({ menuItems, selectedId, menuLabel, updateState }) => {
     updateState(id, title);
     const selector = `a[name="${id}"]`;
     smoothScroll(null, selector, 50);
+    triggerFocus(selector);
   };
 
+  /**
+   * Trigger the focus on screen readers, so they can read the target paragraph
+   *
+   * @param {*} elem Selector to find the item
+   */
+  function triggerFocus(elem) {
+    const element = document.querySelector(elem);
```

Comment:
Can use `root` here to prevent `undefined` in SSR environments.

e.g. https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/1726663f273fc08429c442b91d061d7a967afb02/packages/react/src/components/Masthead/Masthead.js#L27

Reply:
Forgive my ignorance, but I'm not familiar with this yet. I will keep looking through this and other examples to see if I can pick it up, but if you can spare any extra explanation that would be super helpful. :)

Reply:
Resolved separately on slack.